pennylane/resource/resource.py

```diff
@@ -125,6 +182,177 @@ def resources(self) -> Resources:
         """
 
 
+def add_in_series(r1: Resources, r2: Resources) -> Resources:
```

Comment:
The initial names were first and second. Any reason for changing them here?

Reply:
These were the names used in [sc-77943]